articles/api-auth/dynamic-client-registration.md

```diff
@@ -0,0 +1,236 @@
+---
+description: How to dynamically register Clients with Auth0, using the Management APIv2.
+toc: true
+---
+
+# Dynamic Client Registration
+
+Dynamic Client Registration is a feature, based on the [OpenID Connect Dynamic Client Registration specification](https://openid.net/specs/openid-connect-registration-1_0.html), that enables you to register clients dynamically. These clients are called **Third Party Clients** (since they are registered and used by third party developers).
+
+## Third Party Client characteristics
+
+Third Party Clients have the following characteristics:
+
+- The [ID tokens](/tokens/id-token) generated for these clients, hold minimum user profile information.
+
+- They can use only tenant level connections (domain connections). These are sources of users, configured in the tenant's [dashboard](${manage_url}) as connections, but they are not enabled for any client.
```

Comment:
> they are not enabled for any client.

I think we are mentioning that they don't need to be explicitly enable for a particular client. When I read this tho, it sounds like this type of connection cannot be used by any of the clients.

Maybe rephrase? Make it clear that Domain Connection is enable for every third party client and that optionally it can be enabled for selected first party (standard) clients.